aviatrix/resource_aviatrix_remote_syslog_test.go

```diff
@@ -0,0 +1,68 @@
+package aviatrix
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-aviatrix/goaviatrix"
+)
+
+func TestAccAviatrixRemoteSyslog_basic(t *testing.T) {
```

Comment:
Make sure to run the acceptance tests yourself, this test should have caught the `import` functionality not working.